### ISSUE TITLE
Panel-hide keyboard shortcut doesn't work in Firefox on Windows

### ISSUE THREAD

**Issue by rpj** [robinsloan/the-oakland-follower-sentinel#1]
Apparently Firefox doesn't set `metaKey` properly 🤦:

> Yes, Firefox can detect the Windows key, but not as the "Meta" key. Both keyup and keydown events are fired for that key, but not a keypress event. Chrome also fires events similarly (no keypress). The keyCode is the same on both Chrome and Firefox, but what they consider the key to be is different between the two. Chrome considers the key to be the "Meta" key, while Firefox considers it to be the "OS" key. Thus, Firefox does not set the metaKey property to true, because from Firefox's point of view, the "Meta" key is not being pressed.

-https://stackoverflow.com/a/39299284

As hinted by the above, Chrome behaves "as expected" on Windows, where Shift-Windows-Y hides the publishing panel.

Extremely nit-picky I admit, so no expectations, but thought you'd like to know. 👍 🙇

**Comment by robinsloan** [comment]
Here is an absolutely noob question: what's the natural/preferred command key combo on Windows? Is it ctrl-shift-<key>? Something else?

(Asking because I can just wire up an additional/alternative shortcut.)

**Comment by rpj** [comment]
Not a noob question at all! In fact I only started using Windows during the pandemic to play AAA games the way they're meant to be played, but I love the keyboard on that laptop so find myself using it more often than expected. 

Accordingly I'm no expert on Windows key combos but from what I've seen, Ctrl+Shift is quite common indeed. That would work great.

Thanks a ton for taking a look into this!

**Comment by rdmurphy** [comment]
Sorry to throw another browser annoyance on top of this, but there's a similar issue in MacOS Microsoft Edge — it just straight up claims `Command+Shift+Y` for it's own Collections feature. 😐 I tried to find a way to unset that but had no luck. I don't use Edge as my main browser (was just making sure `<spring-board>` was working over there) so I'm personally not too inconvenienced but could possibly come up for someone else later.

<img width="314" alt="image" src="https://user-images.githubusercontent.com/419145/177067140-395bcd59-6bca-4876-a666-123ff0bc02d1.png">

**Comment by robinsloan** [comment]
Hahahahaha, IS NO KEY COMBINATION AVAILABLE?

I'll have to hunt around for something sufficiently obscure 😝

**Comment by rdmurphy** [comment]
I was thinking about this again and while it wouldn't address the keyboard combo issue, it may be handy to have an actual toggle on the page somewhere to flip this on and off. I've often wished it was something that was also tracked via local storage! Sometimes I'm strictly in "read mode" and don't wanna see the editor by default.

**Comment by robinsloan** [comment]
Yes! I agree. My current thinking (and I have been sooo delinquent getting this shipped) is that the editor should actually live on a separate page. Easy to reach -- totally fast and snappy -- but a wholly different view rather than "a panel". This way I'll have space for a live preview, as well.
